pkg/solana/chain.go

```diff
@@ -292,11 +292,11 @@ func newChain(id string, cfg *config.TOMLConfig, ks loop.Keystore, lggr logger.L
 	tc := func() (client.ReaderWriter, error) {
 		return ch.getClient()
 	}
-	ch.txm = txm.NewTxm(ch.id, tc, cfg, ks, lggr)
+	ch.txm = txm.NewTxm(ch.id, tc, cfg, ch.multiNode, ks, lggr)
```

Comment:
Fair point, we don't need to pass in the multiNode but we still need to know if it's enabled. We can pass in a TOMLConfig which contains the multinode config and use that to check instead. 

If MultiNode is enabled, the TXM will use the passed in `tc` function directly, instead of only using it with the lazy loader (which would negate the proper use of MultiNode).